spec/factories/translations.rb

```diff
@@ -0,0 +1,42 @@
+FactoryGirl.define do
+  factory :translation, aliases: [:project_translation] do
+    strings({
+      title: "A test Project",
+      description: "Some Lorem Ipsum",
+      introduction: "Good times intro",
+      workflow_description: "Go outside",
+      researcher_quote: "This is my favorite project",
+      urls: [
+        {label: "Blog", url: "http://blog.example.com/"},
+        {label: "Twitter", url: "http://twitter.com/example"}
+      ]
+    })
+    association :translated, factory: :project
+    language "en-GB"
+
+    factory :workflow_translation do
```

Comment:
Is this factory meant to be nested inside?

Reply:
yeah - if you use the nesting it inherits the attributes of the outer scope.